src/main/java/ui/MenuControl.java

```diff
@@ -87,7 +87,28 @@ private Menu createFileMenu() {
         return file;
 
     }
-    
+
+    private void onBoardNewBoard() {
+        logger.info("Menu: Boards > New");
+
+        List<PanelInfo> panelList = panels.getCurrentPanelInfos();
+        if (!panelList.isEmpty()) {
+            Alert dlg = new Alert(AlertType.CONFIRMATION, "");
+            dlg.initModality(Modality.APPLICATION_MODAL);
+            dlg.setTitle("Confirmation");
+            dlg.getDialogPane().setHeaderText("Create a new board");
```

Comment:
It'd be nice for SLAP if all this could be factored out into a method, maybe named `confirmNewBoardCreation`. It could return a boolean value which tells us whether we can safely close everything and save, etc.